File: 2016_10_28/bubble_sort/bubble_sort.py
#!/usr/bin/python3

import random

def bubble_sort(array):
	for i in range(len(array)):
		for j in range(1, len(array) - i):
			if array[j] < array[j - 1]:
				array[j], array[j - 1] = array[j - 1], array[j]
			#print(array)









test = []
for i in range(20):
	test.append(i)

random.shuffle(test)

print(test)
bubble_sort(test)
print(test)

File: 2016_10_28/bubble_sort/bubble_sort.js
#!/usr/bin/node

function bubble_sort(a) {
	for (i in a)
		for (let j = 1; j <= a.length - i; j++)
			if (a[j] < a[j - 1])
				[a[j], a[j - 1]] = [a[j - 1], a[j]];
}

let a = [];



























